<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>千麦报告单状态</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSQLdata">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LIS数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushQMdata">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>千麦数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushQuit">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpBoxStartDate">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>开始日期</string>
    </property>
    <widget class="QDateEdit" name="dateStartDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxEndDate">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>结束日期</string>
    </property>
    <widget class="QDateEdit" name="dateEndDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
